pandas/tests/io/json/test_ujson.py

```diff
@@ -218,10 +198,7 @@ def test_encode_array_of_nested_arrays(self):
         assert nested_input == json.loads(output)
         assert nested_input == ujson.decode(output)
 
-        nested_input = np.array(nested_input)
-        tm.assert_numpy_array_equal(
-            nested_input, ujson.decode(output, numpy=True, dtype=nested_input.dtype)
-        )
+        assert nested_input == ujson.decode(output)
```

Comment:
Looks like this is already asserted above.

Reply:
Hm weird, thx. Removed